Doc/library/struct.rst

```diff
@@ -144,12 +151,12 @@ following table:
 
 If the first character is not one of these, ``'@'`` is assumed.
 
-Native byte order is big-endian or little-endian, depending on the host
-system. For example, Intel x86 and AMD64 (x86-64) are little-endian;
-IBM z and most legacy architectures are big-endian;
-and ARM, RISC-V and IBM Power feature switchable endianness
-(bi-endian, though the former two are nearly always little-endian in practice).
-Use ``sys.byteorder`` to check the endianness of your system.
+Native byte order is big-endian or little-endian, depending on the
+host system. For example, Intel x86, AMD64 (x86-64), and Apple M1 are
+little-endian; IBM z and most legacy architectures are big-endian; and
+ARM, RISC-V and IBM Power feature switchable endianness (bi-endian,
```

Comment:
I'd get rid of the entire "; and ARM ... switchable ... though in practice)." sentence as it isn't important.

Reply:
Will do.